CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Template
 
+- more update ([#2786](https://github.com/nf-core/tools/pull/2786))
```

Comment:
I feel like we're missing changelog from https://github.com/nf-core/tools/pull/2785

Reply:
yes, because it was open before we merged #2783